submissions/2024-10-30-OpenFF-Lipid-Optimization-Benchmark-Supplement-v1.0/opt.toml

```diff
@@ -0,0 +1,64 @@
+name = "OpenFF Lipid Optimization Benchmark Supplement v1.0"
+description = """
+
+An optimization data set created to improve the training coverage of lipid-like
+molecules in Sage. The molecules in this data set were selected from the [LIPID
+MAPS](https://www.lipidmaps.org/) database via
+[cura](https://github.com/ntBre/curato/tree/6039bea5c64f8cd6b374fd12b5fa3b355898d98b)
+after being clustered on the 2048-bit, radius-3 Morgan fingerprint from RDKit by
+the `LeaderPicker.LazyBitVectorPick` algorithm, also from RDKit, with a distance
+threshold of 0.705. The candidate molecules were further restricted to those
```

Comment:
It looks like `main.py` sets 0.708 -- should one of these be updated?

Edit: looks like the README matches main.py -- but is the dataset description also created from opt.toml?